Update libgav1 to fix AArch64 builds?

I had to disable NEON in libgav1 due do this build error:

```
external/libgav1/src/dsp/arm/film_grain_neon.cc:756:40: error: cannot convert 'const int32x4_t' to 'int16x8_t'
  756 |     noise = ScaleNoise(noise, scaling, scaling_shift_vect32);
      |                                        ^~~~~~~~~~~~~~~~~~~~
      |                                        |
      |                                        const int32x4_t
```

And it looks that it is fixed already:

```
commit 0a72a9f888f166396d4a3c9228a7b7299f61b953
Author: Alex Peterson <petersonab@google.com>
Date:   Thu Jul 15 16:28:07 2021 -0700

    Use vqrdmulhq in ScaleNoise in film_grain_neon.cc
    
    This avoids lengthening to int32x4_t
    
    PiperOrigin-RevId: 385033613
    Change-Id: I694a31b0dd52271d7cd04bef1b60c8598578f1e7
```

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -521,7 +521,7 @@ http_archive(
 new_git_repository(
     name = "libgav1",
     build_file = "//third_party:libgav1.BUILD",
-    commit = "6ab7d65a68350ed4ec6aaabfa18715b2d76a231c",
+    commit = "07a59c59d4d180d67ea0ae5269e3c092c87286e5",
     remote = "https://chromium.googlesource.com/codecs/libgav1",
 )
 

